JustBrewIt/Pages/Brewery.cshtml.cs

```diff
@@ -13,14 +13,22 @@ public class BreweryModel : PageModel
     {
         public object MyBreweryAPI { get; set; }
 
+        public string Error;
         public void OnGet()
         {
-
             using (var webClient = new WebClient())
             {
-                string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries?by_city=cincinnati&brewery_type=regional");
-                Welcome[] welcome = Welcome.FromJson(jsonString);
-                ViewData["MyBreweryAPI"] = welcome;
+                try
+                {
+                    string breweryResponseString = webClient.DownloadString("https://api.openbrewerydb.org/breweries?by_city=cincinnati&brewery_type=regional");
```

Comment:
Modified Variable name to breweryResponseString, to make it more relevant
Reference:- https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Reply:
Good.